kubernetes/kind/create.sh

```diff
@@ -76,10 +75,11 @@ $KUBECTL --namespace default wait MongoDBCommunity/mongodb \
     --for=jsonpath='{.status.phase}'=Running \
     --timeout=10m
 
-# Change directory back to the original directory
-cd -
-
 # Download the TLS certificate and key from the TLS secret created by Cert Manager. Decode them with
 # jq and concatenate them together to match the format expected by MongoDB drivers.
 $KUBECTL --namespace default get secret mongodb-tls -o json | \
     $JQ -r '.data."tls.crt", .data."tls.key" | @base64d' > mongodb_tls_cert.pem
+
+# Sleep for 30 seconds to allow the Kubernetes operator to add the user to the database.
+# TODO: Find a better way to wait for the availability of the user than just sleeping.
+sleep 30
```

Comment:
Use `mongo` shell to send a command requiring authentication with a small server selection and socket timeouts?

Reply:
Good idea, I'll give that a shot.

Reply:
Updated to use `mongosh` in a loop to test that connecting with the expected userinfo works.